spec/postgresql_upgrade_hook_context_extension_spec.rb

```diff
@@ -70,12 +70,12 @@
 
     it 'runs postgresql-setup --upgrade' do
       expect(subject).to be_nil
-      expect(context).to have_received(:'execute!').with("runuser -l postgres -c 'PGSETUP_INITDB_OPTIONS=\"--lc-collate=en_US.UTF-8 --lc-ctype=en_US.UTF-8 --locale=en_US.UTF-8\" postgresql-setup --upgrade'", false, true)
+      expect(context).to have_received(:'execute!').with("runuser -l postgres -c 'PGSETUP_INITDB_OPTIONS=\"--lc-collate=en_US.UTF-8 --lc-ctype=en_US.UTF-8 --locale=en_US.UTF-8\" postgresql-setup --upgrade'", false, true, {})
```

Comment:
`runuser -l` clears the environment, so that's why I did *not* use the env stuff here too.